doc/libfbdoc/fbdoc_keywords.bas

```diff
@@ -79,7 +79,13 @@ namespace fb.fbdoc
 
 		keyworddata = LoadFileAsString( sFileName )
 
-		keywordhash.alloc( 1000 )
+		keywordhash.alloc( 2000 )
+
+		if( keyworddata = "" ) then
+		exit function
```

Comment:
This `exit function` is probably not supposed to be there

Reply:
Correct.  I did get the printed message at one point during testing... weird.  Thanks.